config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root to: 'pages#home'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :pages do
+    collection do
+      get 'entries'
```

Comment:
Let's use symbols like:
```suggestion
      get :entries
```
and so on...